### DISCUSSION TITLE
Github Student developer pack approved but benefits are not available yet.

### DISCUSSION THREAD

**Discussion by ahaque98** [community/community#110212] [GitHub Education] [answered]
### Hi everyone,

Hello!

My Github Student Developer Pack was approved 2 days ago but till now i have not received any emails regarding the benefits. 

![image](https://github.com/community/community/assets/65084644/b924aca3-f54c-403d-959d-cc1964dfb165)

How long does it usually take to recieve the benefits?

**Comment by DaniAffCH** [comment]
It took 2 days for me.
An [old discussion ](https://github.com/orgs/community/discussions/64291) seems to confirm

**Reply by ahaque98** [reply]
Ah I see. Didn't know there was a discussion about it. 

You are lucky as It has already been two days for me and I still haven't received the benefits .

**Reply by DaniAffCH** [reply, marked as answer]
The answer from the discussion specifies that takes 1-2 **business days**. So Sunday doesn't count. Keep waiting, you will receive your benefits soon.

**Comment by Jinansh230705** [comment]
Same here ,
I had applied first but it got rejected because I didn't setup 2FA 
After resolving all issues, i applied again and it was approved but I still haven't got benefits.
